src/ApplicationCore/Entities/FraudProtectionApiModels/Response/AssessmentResponse.cs

```diff
@@ -1,63 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace Contoso.FraudProtection.ApplicationCore.Entities.FraudProtectionApiModels.Response
```

Comment:
I don't mind deleting this, but if we ever need to read the DecisionDetails or some part of the response in order to adjust UI we'll have to re-add something like this (even if it's much smaller).  I think this also makes it a a bit harder for customers to use as a reference implementation as they'll need to generate this class themselves (or again at least a small part of it). What about keeping the API as Object, but keeping at least part of this contract around for reference and possible future use?

Reply:
yeah, i have no objections on that.

i can re-add this file and also add the additional `enrichments` properties to this type so that it's visible to customers.